cassandra-mesos-model/src/main/java/io/mesosphere/mesos/util/CassandraFrameworkProtosUtils.java

```diff
@@ -339,7 +339,7 @@ public ResourceHasExpectedRole(@NotNull final String role) {
 
         @Override
         public boolean apply(final Resource item) {
-            return item.getRole().equals(role);
+            return "*".equals(role) || "*".equals(item.getRole()) || item.getRole().equals(role);
```

Comment:
I don't think we want to absorb the extra logic necessary to be able to combine offers right now to the point that we could use resources across multiple roles.  Please remove this for now, we can decide if we want to do something around multi-role support later on.